components/prod/c100-application.tf

```diff
@@ -0,0 +1,12 @@
+data "local_file" "c100-application-config" {
+  filename = "${path.cwd}/../../environments/prod/apply-to-court-about-child-arrangements-service-justice-gov-uk.yml"
```

Comment:
this file does not exist

Reply:
Was waiting on merge from https://github.com/hmcts/azure-public-dns/pull/942